packages/common/src/search/_internal/hubSearchItemsHelpers/ogcItemToDiscussionPostResult.ts

```diff
@@ -0,0 +1,26 @@
+import { IOgcItem } from "./interfaces";
+import { IPost } from "../../../discussions";
+import { IHubSearchResult } from "../../types";
+
+export async function ogcItemToDiscussionPostResult(
```

Comment:
can you add some comments above this - focus on the "why" vs "how"

Reply:
Since the coverage is 100%, this is ok, but I'd encourage having tests for these internal helpers vs depending on complex higher-level tests... if the higher level test fails, it's often not clear if the issue is in the top level function, or deeper in the stack. :)